### PR TITLE
Rebuild 10.0 (s390x)

### PR DESCRIPTION
**Pull request by ELundby45**
# Rebuild humanfriendly 10.0

Rebuild in order to add support for s390x

- Bump build number
- Update about section

`optimum` -> `coloredlogs` -> `humanfriendly`

### CODE REVIEW

**Review by AndrewVallette — APPROVED** [review]
LGTM!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -38,16 +38,17 @@ test:
...
   license_family: MIT
   license_file: LICENSE.txt
+  license_url: https://github.com/xolox/python-humanfriendly/blob/master/LICENSE.txt
   summary: 'Human friendly output for text interfaces using Python.'
   description: |
     The functions and classes in the humanfriendly package can be used to make text interfaces more user friendly. 
   doc_url: https://humanfriendly.readthedocs.io/en/latest/
   dev_url: https://github.com/xolox/python-humanfriendly
-  doc_source_url: https://github.com/xolox/python-humanfriendly/tree/master/docs
+  doc_source_url: https://github.com/xolox/python-humanfriendly/tree/master/docs
```
Comment (katietz) [inline recipe/meta.yaml]: there is a doc_url, please don't add superflous doc_source_url

File: recipe/meta.yaml [hunk 2]
```diff
@@ -38,16 +38,17 @@ test:
     - humanfriendly --help  # [not unix]
 
 about:
-  home: https://humanfriendly.readthedocs.org
+  home: https://humanfriendly.readthedocs.io
   license: MIT
   license_family: MIT
   license_file: LICENSE.txt
+  license_url: https://github.com/xolox/python-humanfriendly/blob/master/LICENSE.txt
```
Comment (katietz) [inline recipe/meta.yaml]: the license file is in the package. no need (even dangerous in terms of correctness) to add here this superflous license_url on top. please stop this habbit

**Review by katietz — CHANGES_REQUESTED** [review]

**Review by varlackc — APPROVED** [review]
LGTM

**Pull request merged**